com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -401,6 +401,8 @@ private SnapshotDespawnCommand GetDespawnCommand()
             var command = new SnapshotDespawnCommand();
```

Comment:
I take it we shouldn't pass the networkObj in the xtor?  We sometimes want to instantiate without a network object, yes?